src/app/pages/examiner-stats/examiner-stats.analytics.effects.ts

```diff
@@ -0,0 +1,130 @@
+    ofType(ExaminerStatsViewDidEnter),
+    switchMap(() => {
+      this.analytics.setCurrentPage(AnalyticsScreenNames.EXAMINER_STATS);
+      return of(AnalyticRecorded());
+    }),
+  ));
+
+  dateRangeChanged$ = createEffect(() => this.actions$.pipe(
+    ofType(DateRangeChanged),
+    switchMap(({ range }) => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.DATE_RANGE_CHANGED,
+        range,
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+
+  locationChanged$ = createEffect(() => this.actions$.pipe(
+    ofType(LocationChanged),
+    switchMap(({ location }) => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.LOCATION_CHANGED,
+        location,
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+
+  testCategoryChanged$ = createEffect(() => this.actions$.pipe(
+    ofType(TestCategoryChanged),
+    switchMap(({ testCategory }) => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.TEST_CATEGORY_CHANGED,
+        testCategory,
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+
+  colourFilterChanged$ = createEffect(() => this.actions$.pipe(
+    ofType(ColourFilterChanged),
+    switchMap(({ colour }) => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.COLOUR_SCHEME_CHANGED,
+        colour,
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+
+  accordionOpened$ = createEffect(() => this.actions$.pipe(
+    ofType(AccordionOpened),
+    switchMap(() => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.ADDITIONAL_FILTERS_TOGGLED,
+        'Additional filters opened',
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+  accordionClosed$ = createEffect(() => this.actions$.pipe(
+    ofType(AccordionClosed),
+    switchMap(() => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.ADDITIONAL_FILTERS_TOGGLED,
+        'Additional filters closed',
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+
+  hideChartsEnabled$ = createEffect(() => this.actions$.pipe(
+    ofType(HideChartsActivated),
+    switchMap(() => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.HIDE_CHARTS_CHANGED,
+        'Charts hidden',
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
+  hideChartsDisabled$ = createEffect(() => this.actions$.pipe(
+    ofType(HideChartsDeactivated),
+    switchMap(() => {
+      this.analytics.logEvent(
+        AnalyticsEventCategories.EXAMINER_STATS,
+        AnalyticsEvents.HIDE_CHARTS_CHANGED,
+        'Charts unhidden',
+      );
+      return of(AnalyticRecorded());
+    }),
+  ));
```

Comment:
This should have been left as was, and instead of two different analytics just
```
this.analytics.logEvent(
        AnalyticsEventCategories.EXAMINER_STATS,
        AnalyticsEvents.HIDE_CHARTS_CHANGED,
        hideCharts ? 'Charts hidden' : 'Charts unhidden',
);
```